draft-ietf-quic-recovery.md

```diff
@@ -132,7 +132,8 @@ the encryption level and includes a packet sequence number (referred to below as
 a packet number).  The encryption level indicates the packet number space, as
 described in {{QUIC-TRANSPORT}}.  Packet numbers never repeat within a packet
 number space for the lifetime of a connection.  Packet numbers are sent in
-monotonically increasing order within a space, preventing ambiguity.
+monotonically increasing order within a space, preventing ambiguity.  It is
+permitted for some packet numbers to never be used, leaving intentional gaps.
```

Comment:
Do we now also have to acknowledge the fact that the loss recovery algorithms assume the absence of gaps?

Reply:
The algorithm does not assume the absence of gaps -- Section 6.1 says: `The packet was sent kPacketThreshold packets before an acknowledged packet`. The pseudo-code does assume it though, and where it does, it is stated in a comment (see A.10).

All in all, I think this addition is safe.

Reply:
My expectations matched what Jana describes.